src/TwitchLib.Communication/Enums/WhisperRateLimit.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace TwitchLib.Communication.Enums
+{
+    /// <summary>
+    ///     <see href="https://discuss.dev.twitch.tv/t/whisper-rate-limiting/2836"/>
+    ///     <br></br>
+    ///     see also <see cref="IClientOptions.ThrottlingPeriod"/>:
+    ///     <br></br>
+    ///     <inheritdoc cref="IClientOptions.ThrottlingPeriod"/>
+    /// </summary>
+    public enum WhisperRateLimit : uint
+    {
+        // to have only one Reset-Timer running that fires each 30 seconds,
+        // this value is set to 50
+        // please take a look at this enums summary
+        Limit_100_in_60_Seconds = 50
+    }
+}
```

Comment:
Whispers are nolong in IRC and this is not needed.

Reply:
May/Should i remove all stuff related to whispers?

Reply:
If you're going to remove it, I would remove it only from this PR. If there is more whispers code in the library, it can get messy, and it would be better to have it in a separate PR.

Reply:
Yeah. I got it.
I'm going to remove the internal stuff and to mark the public ones as obsolete.